components/user_profile/user_profile_edit.js

```diff
@@ -0,0 +1,118 @@
+      gameTags: ['game1', 'game2', 'game3', 'game4'],
+      discord: 'DummyDiscord',
+      steam: 'DummySteam',
+      battlenet: 'DummyBattle.net'
+    }
+  }
+
+  render () {
+    let styles = StyleSheet.create({
+      title: {
+        fontSize: 12
+      },
+      userValues: {
+        fontSize: 24
+      }
+    })
+    const { navigate } = this.props.navigation
+
+    return (
+      <Container>
+        <Header>
+          <Left>
+            <Button transparent>
+              <Icon name='menu' />
+            </Button>
+          </Left>
+          <Body>
+            <Title>Edit Profile</Title>
+          </Body>
+          <Right>
+            <Button full rounded light onPress={() => navigate('UserProfile')}>
+              <Text fontSize={4}>
+              Back
+            </Text>
+            </Button>
+          </Right>
+        </Header>
+        <Content>
+          <Body>
+            <Text style={styles.userValues}>
+              {'\n'}
+            </Text>
+            {/* Thumbnail for Profile Picture */}
+            <Thumbnail large source={this.state.profilePic} />
+            {/* /* Display Username */}
+            <Text style={styles.title}>
+              {'\n'}
+            Update Username
+          </Text>
+            <Item style={styles.userValues}>
+              <Input onChangeText={(input) => this.setState({username: input})} />
+            </Item>
+            {/* /* Display Email */}
+            <Text style={styles.title}>
+              {'\n'}
+              Update Email
+            </Text>
+            <Item style={styles.userValues}>
+              <Input onChangeText={(input) => this.setState({email: input})} />
+            </Item>
+            {/* /* Display Password */}
+            <Text style={styles.title}>
+              {'\n'}
+            Update Password
+          </Text>
+            <Item style={styles.userValues}>
+              <Input onChangeText={(input) => this.setState({password: input})} />
+            </Item>
+          <Text style={styles.title}>
+            {'\n'}
+            Discord
+          </Text>
+          <Item style={styles.userValues}>
+            <Input onChangeText={(input) => this.setState({discord: input})} />
+          </Item>
+            <Text style={styles.title}>
+              {'\n'}
+              Steam
+            </Text>
+            <Item style={styles.userValues}>
+              <Input onChangeText={(input) => this.setState({steam: input})} />
+            </Item>
+            <Text style={styles.title}>
+              {'\n'}
+              Battle.net
+            </Text>
+            <Item style={styles.userValues}>
+              <Input onChangeText={(input) => this.setState({battlenet: input})} />
+            </Item>
+          <Text style={styles.title}>
+            {'\n'}
+          </Text>
+          <Button block info>
```

Comment:
This button should activate a 'handleSubmit', which doesn't have to do anything but should be present.